### ISSUE TITLE
No base16-tomorrow-dark in base16-theme pacakge

### ISSUE THREAD

**Issue by thepreacher** [syl20bnr/spacemacs#6903]
<PLEASE REPLACE ALL UPPERCASE EXPRESSIONS AND DELETE THIS LINE>
#### Description
There is no base16-tomorrow-dark theme in the base16-theme package.

#### Reproduction guide
- Start Emacs
Add base16-tomorrow-dark to dotspacemacs-themes variable in .spacemacs.
Start or restart emacs

*Observed behaviour:*
Uses a theme with a white background and displays error message in a new buffer below.

*Expected behaviour:*
Expecting editor to use base16-tomorrow-dark theme

#### System Info
- OS: darwin
- Emacs: 25.1.1
- Spacemacs: 0.105.22
- Spacemacs branch: master (rev. 9f9faa4)
- Graphic display: t
- Distribution: spacemacs
- Editing style: vim
- Completion: helm
- Layers:
```elisp
((auto-completion :variables auto-complete-enable-snipptes-in-popup t)
 better-defaults osx emacs-lisp
 (git :variables git-enable-github-support t git-gutter-use-fringe t)
 markdown
 (org :variables org-enable-github-support t)
 github
 (shell :variables shell-default-height 30 shell-default-position 'bottom shell-default-term-shell "/usr/local/bin/fish")
 spell-checking syntax-checking version-control company-mode html javascript python rust smex colors erlang elixir scala nginx yaml sql react clojure php
 (c-c   :variables c-c  -enable-clang-support t))
```

#### Backtrace
```
BACKTRACE IF RELEVANT
```

**Comment by cpaulik** [comment]
It seems that it is called/was renamed to `base16-tomorrow-night` (see https://github.com/belak/base16-emacs/blob/master/base16-tomorrow-night-theme.el) which is not yet supported by the spacemacs core-themes.el

**Comment by TheBB** [comment]
Should work now in develop.

**Comment by belak** [comment]
Sorry about this. Here's a copied response from me on from the report I got in my package:

```
So, it looks like this is partially my fault and partially just a result of how spacemacs does things.

I have a PR against spacemacs to fix all the themes currently in this package (syl20bnr/spacemacs#6905), but you will still have to change the name of the theme manually once that goes in.

Also, this appears to be related to chriskempson/base16#42. A while back, the set of base16 schemes changed and the builder changed to not recommend building both a light and dark variant (as not all schemes look good with both a light and a dark version) so it was decided to specifically curate those. base16-tomorrow-night does take the place of base16-tomorrow-dark. And base16-tomorrow takes the place of base16-tomorrow-light.

...

I'll leave this open until the spacemacs changes go in. Thanks for the report and sorry for the inconvenience!
```

**Comment by d12frosted** [comment]
Fixed with released of Spacemacs v0.200. Let us know if you still have any problems with this issue.
